BUILD.gn

```diff
@@ -6,56 +6,62 @@ group("flutter") {
   testonly = true
 
   deps = [
-    "//flutter/lib/snapshot:compile_platform",
-    "//flutter/lib/snapshot:generate_snapshot_bin",
-    "//flutter/sky",
-    "//flutter/third_party/txt",
+    "//$flutter/lib/snapshot:compile_platform",
+    "//$flutter/lib/snapshot:generate_snapshot_bin",
+    "//$flutter/sky",
+    "//$flutter/third_party/txt",
   ]
 
   if (!is_fuchsia) {
     if (current_toolchain == host_toolchain) {
       deps += [
         "//dart:create_sdk",
-        "//flutter/frontend_server",
+        "//$flutter/frontend_server",
       ]
     }
   }
 
   if (is_fuchsia) {
     deps += [
-      "//flutter/content_handler",
-      "//flutter/content_handler:aot_content_handler",
-      "//flutter/examples",
-      "//flutter/flow",
+      "//$flutter/content_handler",
+      "//$flutter/content_handler:aot_content_handler",
+      "//$flutter/examples",
+      "//$flutter/flow",
     ]
   }
 
   # If on the host, compile all unittests targets.
   if (current_toolchain == host_toolchain) {
     if (is_mac) {
-      deps += [ "//flutter/shell/platform/darwin:flutter_channels_unittests" ]
+      deps += [ "//$flutter/shell/platform/darwin:flutter_channels_unittests" ]
     }
     if (!is_win) {
-      deps += [ "//flutter/shell/platform/embedder:flutter_engine" ]
+      deps += [ "//$flutter/shell/platform/embedder:flutter_engine" ]
     }
     deps += [
-      "//flutter/flow:flow_unittests",
-      "//flutter/fml:fml_unittests",
-      "//flutter/sky/engine/wtf:wtf_unittests",
-      "//flutter/synchronization:synchronization_unittests",
-      "//flutter/third_party/txt:txt_benchmarks",
-      "//flutter/third_party/txt:txt_unittests",
+      "//$flutter/flow:flow_unittests",
+      "//$flutter/fml:fml_unittests",
+      "//$flutter/sky/engine/wtf:wtf_unittests",
+      "//$flutter/synchronization:synchronization_unittests",
+      "//$flutter/third_party/txt:txt_benchmarks",
+      "//$flutter/third_party/txt:txt_unittests",
       "//garnet/public/lib/fxl:fxl_unittests",
     ]
   }
 }
 
+config("config") {
```

Comment:
I had to define this config and add it to various `source_set`s for includes to work.